Standard

Dulow187 opened the pull request.

josephfrazier committed:
Update README for standardx

josephfrazier committed:
Merge branch 'standardx'

josephfrazier committed:
1.0.2

josephfrazier committed:
Ensure tests pass before publishing

josephfrazier committed:
Handle projects without package.json

josephfrazier committed:
Extract doInstall() helper

josephfrazier committed:
Don't install standardx (or uninstall standard) if `--no-install` is passed

josephfrazier committed:
Add test/clone.js from https://raw.githubusercontent.com/standard/standard/dc5d1dacda93de1f7fdd91fc53b75d013bad3f4b/test/clone.js

josephfrazier committed:
test/clone.js: Use standardx instead of standard

josephfrazier committed:
Install devDependencies needed for test/clone.js

josephfrazier committed:
.gitignore ./tmp

josephfrazier committed:
Don't use pkg-up

It messes with /package.json when running test/clone.js

josephfrazier committed:
Add test/clone.js to `test` script

Fixes https://github.com/josephfrazier/standard-eject/issues/1

josephfrazier committed:
Document `--no-install` option

josephfrazier committed:
1.0.3

josephfrazier committed:
Ignore package.json scripts when installing standardx

josephfrazier committed:
1.0.4

josephfrazier committed:
Add .travis.yml to test on latest Node.js

josephfrazier committed:
Change `prepublish` script to `prepublishOnly`

That way, the tests won't run after a local `yarn install`.
See here for details: https://docs.npmjs.com/misc/scripts

josephfrazier committed:
Add Travis badge to README

josephfrazier commented:
Hi @Dulow187, did you open this by accident? It looks like you want to merge the `master` branch into `standardx`, but `standardx` has already been merged into `master`, and the NPM package is published from the `master` branch. I just forgot to delete the `standardx` branch.

josephfrazier committed:
Reset test repos after ejecting and running standardx

This makes it easier to re-run tests without having to delete ./tmp/
or manually reset every repo.

josephfrazier committed:
Run lint on test script too

josephfrazier committed:
Upgrade standardx/standard-packages

Note that dependencies of repositories in standard-packages are hoisted
up, so that we don't have to install them in each repository.

There are still a couple of repositories that fail:

1. napa: Not yet sure how to fix
2. express-restify-mongoose: It doesn't use standard anymore: https://github.com/standard/standard-packages/issues/19

josephfrazier commented:
I'm going to close this so I don't get notifications when I push to master, but feel free to continue the conversation
